nginx-controller/nginx/nginx.go

```diff
@@ -211,9 +239,9 @@ func (nginx *NginxController) Start() {
 	}
 }
 
-func (nginx *NginxController) createCertsDir() {
-	if err := os.Mkdir(nginx.nginxCertsPath, os.ModeDir); err != nil {
-		glog.Fatalf("Couldn't create directory %v: %v", nginx.nginxCertsPath, err)
+func (nginx *NginxController) createDir(path string) {
```

Comment:
should be a function, not a method, since it doesn't depend on NginxController anymore